samcli/commands/sync/command.py

```diff
@@ -260,6 +261,7 @@ def do_cli(
 
     set_experimental(ExperimentalFlag.Accelerate)
     update_experimental_context()
+    EventTracker.track_event("UsedFeature", "Accelerate")
```

Comment:
I would recommend using enums for `UsedFeature` and `Accelerate` values, so that we don't need to repeat same string in other places. And it will also eliminate any possible typo in them.

Reply:
While we do have the enum `EventName`, the method currently only accepts a string as its first parameter. The `Accelerate` value is verified via the `EventType` class, which is cross-referenced upon creation of an Event, as I found that was the cleanest way to keep track of each event's potential values, as well as the simplest way to add on new events in the future. A refactor for `track_event` to take an `EventName` instead is possible, and would require an extra import of the enum, but it may be best to push changes like those in a separate PR since it would affect all (7) currently existing `track_event` calls, as well as its unit test methods.

Reply:
Sure, since this is merging to feature branch we can make those changes later on then.